src/centralserver/admin-ui/src/App.vue

```diff
@@ -0,0 +1,34 @@
+<template>
```

Comment:
All `*.vue` and `*.ts` files are missing the license header

Reply:
I think that could be fixed in XRDDEV-1606 since it will anyway change the admin-ui -- probably even replacing many of the current placeholder files that currently miss the license.

We could also consider removing the separate license-add task from front-end projects and use the same gradle plugin as everything else. It should be capable of handling the ts/js/vue files as well with some additional configuration.

Reply:
Yeah that is probably the right ticket to figure out the license headers and how the gradle license plugin could be utilised for this.